doc/contents/Tablesaw_display_as_TableDisplay.ipynb

```diff
@@ -80,20 +79,18 @@
     "    Displayers.register(Table.class, new Displayer<Table>() {\n",
     "      @Override\n",
     "      public Map<String, String> display(Table table) {\n",
-    "        return new HashMap<String, String>() {{\n",
-    "          new TableDisplay(\n",
-    "                  table.rowCount(),\n",
-    "                  table.columnCount(),\n",
-    "                  table.columnNames(),\n",
-    "                  new TableDisplay.Element() {\n",
-    "                    @Override\n",
-    "                    public String get(int columnIndex, int rowIndex) {\n",
-    "                      return table.get(columnIndex, rowIndex);\n",
-    "                    }\n",
+    "        new TableDisplay(\n",
+    "                table.rowCount(),\n",
+    "                table.columnCount(),\n",
+    "                table.columnNames(),\n",
+    "                new TableDisplay.Element() {\n",
+    "                  @Override\n",
+    "                  public String get(int columnIndex, int rowIndex) {\n",
+    "                    return table.get(columnIndex, rowIndex);\n",
     "                  }\n",
-    "          ).display();\n",
-    "          put(MIMEContainer.HIDDEN.getMime().asString(), MIMEContainer.HIDDEN.getData().toString());\n",
-    "        }};\n",
+    "                }\n",
+    "        ).display();\n",
+    "        return OutputCell.HIDDEN_DISPLAYER;\n",
```

Comment:
OK this looks great except call this `OutputCell.DISPLAYER_HIDDEN`
i know that seems like a detail but with autocomplete our existing users don't want extra stuff to show up.